hpat/pd_series_ext.py

```diff
@@ -423,17 +423,17 @@ def resolve_cov(self, ary, args, kws):
         assert len(args) == 1 and isinstance(args[0], (SeriesType, types.Array))
         assert isinstance(ary.dtype, types.Number)
         assert isinstance(args[0].dtype, types.Number)
-        # TODO: complex numbers return complex
-        return signature(types.float64, *args)
+        ret_typ = types.complex128 if isinstance(args[0].dtype, types.Complex) else types.float64
```

Comment:
I think `ary.dtype` should also be checked since the output is complex if either input is complex.

Reply:
Ah, I see.
So, the `ary` is method receiver and `args[0]` is a first argument, so we have:
```python
cov(ary, args[0]) # Re-written as a func call
```
Is that correct?